server/src/main/java/org/opensearch/action/search/AbstractSearchAsyncAction.java

```diff
@@ -220,6 +221,7 @@ public final void start() {
                     null
                 )
             );
+            searchRequestContext.getSearchRequestOperationsListener().onRequestEnd(searchRequestContext);
```

Comment:
Seems like it was missed previously? 

```suggestion
            onRequestEnd(searchRequestContext);
```

Reply:
Fixed